### PR TITLE
Wg timestamp in node repo

### PR DESCRIPTION
**Pull request by gjoranv**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/NodePatcher.java [hunk 1]
```diff
@@ -274,6 +275,8 @@ private Node applyField(Node node, String name, Inspector value, Inspector root,
                 return nodeWithTrustStore(node, value);
             case "wireguardPubkey":
                 return node.withWireguardPubkey(SlimeUtils.optionalString(value).map(WireguardKey::new).orElse(null));
+            case "wireguardKeyTimestamp":
```
Comment (freva) [inline node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/NodePatcher.java:278]: Why is this patchable? Take the current timestamp in node-repo when the key is updated instead?

Comment (gjoranv) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/NodePatcher.java]: It did not make sense to patch this separately. Updated to setting it along with the key.

Comment (freva) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/NodePatcher.java]: Why should it be patchable at all? Generally we don't take timestamps for client for things that are meant to be 'now'.

Comment (gjoranv) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/NodePatcher.java:278]: I misunderstood your comment. Will fix.

**Review by freva — DISMISSED** [review]

**Review by freva — APPROVED** [review]

**Pull request merged**